src/arm/96boards.c

```diff
@@ -199,7 +205,12 @@ mraa_96boards()
 
     if (mraa_file_exist(DT_BASE "/model")) {
         // We are on a modern kernel, great!!!!
-        if (mraa_file_contains(DT_BASE "/model", "Qualcomm Technologies, Inc. APQ 8016 SBC")) {
```

Comment:
As you've corrected the ordering in `arm.c` per @Mani-Sadhasivam's request, please do the same here, to use the same ordering style.